bpm/bpm_extractor.py

```diff
@@ -32,4 +33,5 @@ def get_song_bpm(filename):
     res = pool.map(get_song_bpm, files)
     pool.close()
 
-    print(tabulate(res, headers=['Song ID', 'BPM', 'Confidence'], tablefmt='orgtbl'))
\ No newline at end of file
+    print(tabulate(res, headers=['Song ID', 'BPM',
+                                 'Confidence'], tablefmt='orgtbl'))
```

Comment:
`print(tabulate(res, headers=['Song ID', 'BPM',
                                 'Confidence'], tablefmt='orgtbl'))`

According to PEP8 I believe the list should be defined like this instead:
`print(tabulate(res, headers = [
                                 'Song ID', 'BPM', 'Confidence'
                                                ], tablefmt = 'orgtbl'))`

Reply:
I think it would be cleaner to not split the array, something like:
``` python
print(tabulate(res, 
               headers=['Song ID', 'BPM', 'Confidence'],
               tablefmt='orgtbl'))
```